Resources/public/mapbender.element.digitizer.js

```diff
@@ -1285,22 +1265,40 @@
         /**
          * Copy feature
          *
-         * @param feature
+         * @param {OpenLayers.Feature} feature
          */
-        copyFeature: function(feature) {
+        copyFeature: function(feature ) {
```

Comment:
The space :)

Reply:
@sveta-wl, this is why you don't agree with this pull-request?